hoomd/write/dcd.py

```diff
@@ -0,0 +1,104 @@
+import hoomd
+from hoomd import _hoomd
+from hoomd.filter import ParticleFilter, All
+from hoomd.data.parameterdicts import ParameterDict
+from hoomd.operation import Writer
+
+
+class DCD(Writer):
+    R""" Writes simulation snapshots in the DCD format
+
+    Args:
+        filename (str): File name to write.
+        trigger (hoomd.trigger.Trigger): Select the timesteps to write.
```

Comment:
```suggestion
        trigger (hoomd.trigger.Periodic): Select the timesteps to write.
```
Document explicitly that only periodic triggers are allowed.